metrics/ddl.go

```diff
@@ -70,13 +70,10 @@ var (
 			Buckets:   prometheus.ExponentialBuckets(0.001, 2, 20), // 1ms ~ 524s
 		}, []string{LblResult})
 
-	OwnerUpdateGlobalVersion    = "update_global_version"
-	OwnerGetGlobalVersion       = "get_global_version"
-	OwnerCheckAllVersions       = "check_all_versions"
-	OwnerNotifyCleanExpirePaths = "notify_clean_expire_paths"
-	OwnerCleanExpirePaths       = "clean_expire_paths"
-	OwnerCleanOneExpirePath     = "clean_an_expire_path"
-	OwnerHandleSyncerHistogram  = prometheus.NewHistogramVec(
```

Comment:
Please remove `StartCleanWork` in [line105](https://github.com/pingcap/tidb/pull/36863/files#diff-e880516abc91d29e0ea2a8619091e50fd31338e2a9fae2d4da360662b8d5cbe1L105)